perf/specjbb/build/getSpecjbbSuite.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+
+SUCCESS=0
+ERROR=1
+
+exists() {
+    local specjbb_dir=$1
+    return $(test -f "${specjbb_dir}/${SPECJBB_JAR}")
```

Comment:
Jhon in other channel mentioned "test" is not returning a value.
I tried on my machine too. Can you please validate return value of $(test -f "${specjbb_dir}/${SPECJBB_JAR}")

Reply:
The test above doesn't work because test -f "${d}" checks for the existence of a file not a directory. When I the script a path to a file it works. In the exists function, I am checking for the existence of the specjbb jar file.